lib/transformer.js

```diff
@@ -162,12 +169,14 @@ class Transformer {
     yield fs.writeFile(this.options.utilPath, utilsText);
 
     let interfaceStub = yield fs.readFile(__dirname + "/output/interfaceStub.js");
-    let keys = Object.keys(this._interfaces).concat(Object.keys(this._dictionaries));
+    let keys = Object.keys(this._interfaces).concat(Object.keys(this._dictionaries)).concat(Object.keys(this._typedefs));
     for (let key of keys) {
       if (this._interfaces[key]) {
         yield fs.writeFile(path.join(outputDir, this._interfaces[key].name + ".js"), interfaceStub);
-      } else {
+      } else if (this._dictionaries[key]) {
         yield fs.writeFile(path.join(outputDir, this._dictionaries[key].name + ".js"), "");
+      } else {
```

Comment:
I guess while here it would be good to change this to `else if (this._typedefs[key])` as that seems a bit less likely to blow up in the future.